test/module/irohad/model/converters/json_commands_test.cpp

```diff
@@ -52,10 +52,10 @@ TEST_F(JsonCommandTest, ClassHandlerTest) {
       std::make_shared<AddPeer>(),
       std::make_shared<AddSignatory>(),
       std::make_shared<CreateAccount>(),
-      std::make_shared<CreateAsset>(),
+      std::make_shared<CreateAsset>("", "", 0),
```

Comment:
Maybe it would be better to initialize members in the default constructor?
https://github.com/hyperledger/iroha/blob/fe02823cb87cd319fbaf5f04a94cad16d155c691/irohad/model/commands/create_asset.hpp#L36

Reply:
Well, it could be better, but, to be honest, I just don't want to touch the old model if possible. Hope the day when it pass away is not too far.